9c-internal-mead/argocd/argocd-cm.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-cm
+  namespace: argocd
+  labels:
+    app.kubernetes.io/name: argocd-cm
+    app.kubernetes.io/part-of: argocd
+data:
+  url: https://argocd-internal.planetarium.network
```

```suggestion
  url: https://argocd-internal-mead.planetarium.network
```

Just in case there's an accident.